brownfield_django/main/views.py

```diff
@@ -57,45 +57,42 @@ def get_queryset(self):
         return queryset
 
 
-class UserViewSet(viewsets.ModelViewSet):
-    '''This is for the main page with the list of the users courses
-    it is not for viewing team users or student users, only users who should
-    see a complete list of instructors are admins'''
-    queryset = User.objects.all()
-    serializer_class = UserSerializer
-
-    def get_queryset(self):
-        queryset = User.objects.all().order_by('username')
-        if self.request.user.profile.is_admin():
-            return queryset
-        else:
-            return queryset.filter(id=self.request.user.id)
-
-
 class DocumentViewSet(viewsets.ModelViewSet):
     queryset = Document.objects.all()
     serializer_class = DocumentSerializer
 
-    def update(self, request, pk=None):
-        document = Document.objects.get(id=pk)
-        if document.visible is True:
-            document.visible = False
-        elif document.visible is False:
-            document.visible = True
-        document.save()
-        return Response(document.visible, status.HTTP_200_OK)
-
     def get_queryset(self):
         '''
         Form Docs: queryset that should be used for list views,
         and that should be used as the base for lookups in detail views.
         '''
         course_pk = self.request.QUERY_PARAMS.get('course', None)
+        doc_pk = self.kwargs.get('pk', None)
+
         if course_pk is not None:
             queryset = Document.objects.filter(course__pk=course_pk)
+            return queryset
```

Comment:
The multiple returns could be eliminated here by changing these conditionals to if ... else and returning the queryset at the end of the function.